src/cycax/cycad/assembly.py

```diff
@@ -36,18 +36,16 @@ def render(self):
             # FIXME: Should just be: `part.render()` the part render should sort out its own stuff.
             name = part["part_no"]
 
-            STLname = "{cwd}/{name}/{name}.stl".format(cwd=self._base_path, name=name)
-            if not os.path.exists(STLname):
-                SCADname = "{cwd}/{name}/{name}.scad".format(cwd=self._base_path, name=name)
-                if not os.path.exists(SCADname):
-                    logging.info("Creating a SCAD file of the pieces of the object.")
+            stl_name = "{cwd}/{name}/{name}.stl".format(cwd=self._base_path, name=name)
```

Comment:
I fixed up variable names, quick work in vim. We should just call part.render() here and do none of the scad/stl file checks and "decoding". But that will happen when the other engines gets added.